package.json

```diff
@@ -32,9 +32,15 @@
         "qs": "6.10.3",
         "react": "17.0.2",
         "react-dom": "17.0.2",
+        "remark": "^13.0.0",
+        "remark-directive": "^1.0.1",
+        "remark-gfm": "^1.0.0",
```

Comment:
Can we lock into the specific version of these packages so that we don't use `^`